source/remediation_runbooks/scripts/CreateAccessLoggingBucket_createloggingbucket.py

```diff
@@ -18,6 +26,7 @@ def create_logging_bucket(event, _):
             "Bucket": event["BucketName"],
             "GrantWrite": "uri=http://acs.amazonaws.com/groups/s3/LogDelivery",
             "GrantReadACP": "uri=http://acs.amazonaws.com/groups/s3/LogDelivery",
+            "ObjectOwnership": "ObjectWriter",
```

Comment:
This is creating the bucket for server access logging from the cloudtrail bucket. When created manually, cloudtrail adds a resource policy to allow `PutObject` from the logging service. So no ACLs.

The reason I went with ACLs instead of policies is that the recommended fix is to use a resource policy which restricts the "source" bucket of the logs, which we will not know for both remediations that need this fix because they create the logging bucket before creating the bucket to be logged. Restructuring the remediations would be considerably more tedious work.